### DISCUSSION TITLE
请求增加统计分析功能

### DISCUSSION THREAD

**Discussion by MDsub** [spencerwooo/onedrive-vercel-index#562] [Ideas]
ie [ackee-tracker](https://github.com/electerious/ackee-tracker)

我尝试将script放到footer中，但受到CROS限制无法使用。希望可以增加功能模块 或者 CROS 的相关设置，实现统计分析功能。

**Comment by spencerwooo** [comment]
如果需要加入 script 到网站里面，那么就需要使用 [next/script](https://nextjs.org/docs/api-reference/next/script) 而不是直接在 footer 注入。

**Reply by spencerwooo** [reply]
比如参考 Next.js 加入 Google Analytics 的操作：https://blog.jarrodwatts.com/track-user-behaviour-on-your-website-with-google-analytics-and-nextjs

**Comment by FmgX** [comment]
> 如果需要加入 script 到网站里面，那么就需要使用 [next/script](https://nextjs.org/docs/api-reference/next/script) 而不是直接在 footer 注入。

能否详细的讲一下呢
